src/GraphQL.SystemTextJson/GraphQL.SystemTextJson.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net5;net6;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net7;net6;net5;netcoreapp3.1</TargetFrameworks>
```

Comment:
Adding net7 there will use the STJ 7.0+, instead of (4.7,2) which is used in case of netstandard2.x target. MS claims performance gain in STJ 7 compared to earlier version.

Reply:
System.Text.Json is not a dependency for .NET Core 3.1+ as it is released as part of the framework.  .NET 7 will use the .NET 6 TFM as it is the closest match.  As such, it should have all of the performance benefits available with .NET 7.

Reply:
Roger. I always assumed. If I build a net 6.0 app  always the STJ latest 6.0.x will be package referenced and not the absolute latest (in this case 7.0.x). However I never tested that, so  I will remove net7 as suggested

Reply:
Removed net7